lib/key-mapper.coffee

```diff
@@ -34,8 +34,11 @@ class KeyMapper
           keyDownEvent.shiftKey = false
         else if translation.alted? && @modifierStateHandler.isAltGr()
           charCode = translation.alted
-          keyDownEvent.altKey = false
-          keyDownEvent.ctrlKey = false
+          # do not remove alt+ctrl states for linux
+          # TODO: what about darwin?
+          if process.platform != 'linux'
```

Comment:
If we reset alt+ctrl modifier states here, keymapper fails to resolve bindings that use ctrl, alt or ctrl+alt.

For example pressing 'ctrl+altgr+ß', what you'd use on german keyboard to produce 'ctrl+\', will fail to resolve to 'ctrl+\' (since we just removed the ctrl modifier) and be interpreted as just '\'.